pegasus/sites.v3/code.org/public/educate/csc.haml

```diff
@@ -149,6 +149,6 @@ theme: responsive_full_width
 
 %section#csc-newsletter-signup{style: "padding-top: 2rem"}
   .wrapper
-    = view :curriculum_subscribe_form, subscribe_form_section_desc: hoc_s(:subscribe_form_section_desc_csc), form_src: "http://go.pardot.com/l/153401/2018-10-02/lzp5jd"
+    = view :subscribe_form, subscribe_form_section_title: hoc_s(:subscribe_form_section_title), subscribe_form_section_desc: hoc_s(:subscribe_form_section_desc_csc), form_height: "375px", form_src: "http://go.pardot.com/l/153401/2018-10-02/lzp5jd"
```

Comment:
Related to [this comment](https://github.com/code-dot-org/code-dot-org/pull/55039/files#r1402533719).